src/documentation/app/layout/content/Component.jsx

```diff
@@ -49,26 +49,39 @@ const ExampleWrapper = style.div`
 `;
 
 const ComponentExample = ({
-  name, folder, id,
-}) => [
-  <h4 key="heading">
-    {folder[0] ? `${name} ${folder[0].title}s` : ''}
-  </h4>,
-  <ExampleWrapper key="example">
-    {folder[0]
-      // if this component example contains subfolders, then get example from each subfolder
-      ? folder.map((example, idx) => <Example key={name + idx} example={example} componentName={name} id={id} />)
-      // otherwise just render the example
-      : <Example example={folder} componentName={name} id={id} key={name} />
-    }
-  </ExampleWrapper>,
-];
+  fullscreen, name, folder, id,
+}) => (
+  <Fragment>
```

Comment:
Ah, this is great! I'm always defaulting to using the array `[ ]`, but the `fragments` and `<>` feels like a better approach 👌

Reply:
Yeah totally! Love this feature of React. 😍